src/Microsoft.AspNetCore.Mvc.Core/Internal/MemoryPoolHttpResponseStreamWriterFactory.cs

```diff
@@ -26,10 +26,10 @@ public class MemoryPoolHttpResponseStreamWriterFactory : IHttpResponseStreamWrit
         /// Creates a new <see cref="MemoryPoolHttpResponseStreamWriterFactory"/>.
         /// </summary>
         /// <param name="bytePool">
-        /// The <see cref="ArrayPool{byte}"/> for creating <see cref="byte"/> buffers.
+        /// The <see cref="ArrayPool{Byte}"/> for creating <see cref="byte"/> buffers.
```

Comment:
Ideally we should. But it is not worth the effort as we have a ton of these across all repos..